src/pages/VotePage.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import VoteCell from '../components/VoteCell';
+
+export default function VotePage() {
+  const [candidates, setCandidates] = useState([]);
+
+  async function getCandidates() {
+    // GET, Candidates
+    const response = await axios({
+      method: 'get',
+      url: 'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates',
+      responseType: 'stream',
+    });
+
+    const { status, data } = response;
+
+    switch (status) {
+      case 200:
+        // 투표순위대로 정렬
+        data.sort((a, b) => {
+          return b.voteCount - a.voteCount;
+        });
+        setCandidates(data);
+        break;
+      case 400:
+        alert('권한 없음');
+        break;
+      case 500:
+        alert('서버 오류');
+        break;
+      default:
+        break;
+    }
+  }
+
+  useEffect(() => {
+    getCandidates();
+  });
+
+  const candidatesList = candidates.map((person, index) => {
+    return <VoteCell key={index} vote={getCandidates} {...{ index }} {...{ person }} />;
+  });
```

Comment:
props로 넘길 변수명과 일치할때 비구조화할당해주신점 너무 훌륭하십니당

1. 그런데 `map`함수 자체에서 사용하는 `index`를 변수명 그대로 넘겨주면
`VoteCell`에서 득표순위로 사용하게 될텐데 의미가 불분명하므로 `rank`라는 변수명으로 바꿔서 넘겨주는게 더 좋을 것 같아요~

2. 굳이 `getCandidates`를 넘겨줄 필요가 있었을까요.. ?.?

이렇게 넘겨주셔도 좋을 것 같습니다~~ 이어서 `VoteCell.js`도 확인해주세용:)
```suggestion
  const candidatesList = candidates.map((person, index) => {
    return <VoteCell 
     {
      ...{
        key:index,
        rank:index,
        ...{person},
    }
    }
     />;
  });
```

Reply:
리뷰 감사합니다!!!